### PR TITLE
[WIP] Readme updates

### PR DESCRIPTION
**Pull request by sueann**
WIP PR that needs a bit more work. 

Work so far:
- some minor wordsmithing edits
- a more comprehensive SQL UDF example
- added links to release-specific Databricks notebooks so users know what works in each release. 

Needs:
- a new link to the Databricks notebook for Release 0.3.0 once that notebook has been updated
- reorganize the user guide section in README.md here match the new organization in the 0.3.0 notebook (mostly shuffling the TensorFlow and Keras Transformer sections around) which also adds a TFTransformer example

### CODE REVIEW

**Comment by sueann** [comment]
Done in #106
